default/versioning.py

```diff
@@ -290,6 +295,10 @@ def run(ctx: Context):
     namespace: str = ctx.meta["versioning"]["namespace"]
     scoreholder: str = ctx.meta["versioning"]["scoreholder"]
     version_parts: list[str] = ctx.project_version.split(".")
+    try:
+        scheduled: list[str] = ctx.meta["scheduled"]
+    except:
+        scheduled = []
```

Comment:
instead of `try` and `except` here (bare except I try to avoid).
```py
scheduled: list[str] = ctx.meta.get("scheduled", [])
```

This will return an empty list of `scheduled` if it's not found.

Reply:
Thanks, I've updated the code. Still getting accustomed to how beet works.